apps/test-suite/tests/SQLite.ts

```diff
@@ -97,6 +99,8 @@ CREATE TABLE IF NOT EXISTS test (id INTEGER PRIMARY KEY NOT NULL, name VARCHAR(6
       const result = await db.getFirstAsync<any>('SELECT * FROM translations');
       expect(result.key).toBe('hello');
       expect(result.value).toBe('哈囉');
+
+      await db.closeAsync();
```

Comment:
missing close. on web, we cannot well handle unopened `:memory:` db. it will reuse the cached unopened db and breaks the test